src/metatrain/utils/additive/zbl.py

```diff
@@ -168,7 +170,7 @@ def forward(
         # Set the outputs as the ZBL energies
         targets_out: Dict[str, TensorMap] = {}
         for target_key, target in outputs.items():
-            if target_key.startswith("mtt::aux::"):
+            if is_auxiliary_output(target_key):
```

Comment:
I'm not sure this will be sustainable long term. Should we instead change the API of additive building blocks to explicitly take which outputs they should apply to?

Reply:
I see it as a form of sparsity. We could return `TensorMap`s of zeros, but is that really better? This isn't only happening for these `auxiliary` outputs but also for outputs that are not supported by the additive model (e.g. non-scalars for the composition model or non-energies for ZBL). IMO, the issue with having more options and/or output requests is that model writers will have to take care of that, while right now a model writer can just drop the class in

Reply:
We can improve the documentation though